web/client/src/library/components/editor/EditorTabs.tsx

```diff
@@ -60,7 +60,7 @@ export default function EditorTabs(): JSX.Element {
       addTab(newTab)
     }
 
-    selectTab(newTab)
+    setTimeout(() => selectTab(newTab))
```

Comment:
I understand at a very basic level what this does, but why do this here (setTimeout without a delay)?

Reply:
Forgot to remove. There was a bug where when selecting file in docs and go back to editor, Edito Tab is not selected for selected file. I fixed it [here](https://github.com/TobikoData/sqlmesh/pull/2076)